src/metabase/models/table.clj

```diff
@@ -22,14 +23,23 @@
    (Basically any non-nil value is a reason for hiding the table.)"
   #{:hidden :technical :cruft})
 
+(def ^:const field-orderings
+  "Valid values for `Table.field_order`.
+  `:database`     - use the same order as in the table definition in the DB;
+  `:alphabetical` - order alphabetically by name;
+  `:custom`       - the user manually set the order in the data model
+  `:smart`        - Try to be smart and order like you'd usually want it: first PK, followed by `:type/Name`s, then `:type/Temporal`s, and from there on in alphabetical order."
+  #{:database :alphabetical :custom :smart})
+
 
 (models/defmodel Table :metabase_table)
 
 
 ;;; --------------------------------------------------- Lifecycle ----------------------------------------------------
 
 (defn- pre-insert [table]
-  (let [defaults {:display_name (humanization/name->human-readable-name (:name table))}]
+  (let [defaults {:display_name (humanization/name->human-readable-name (:name table))
+                  :field_order  :database}]
```

Comment:
what's our convention here? Do we simply rely on the DB default or do we set them on the model level as well?

Reply:
it's best to set the default at the DB level if possible. If it's set there than there's no real reason to set it in the model level